src/http-gateways/trustless-gateway.md

```diff
@@ -301,6 +302,32 @@ of their presence in the DAG or the value assigned to the "dups" parameter, as
 the raw data is already present in the parent block that links to the identity
 CID.
 
+## CAR `meta` (content type parameter)
+
+The `meta=eof` parameter allows clients to request the server to include additional metadata about the
+CAR to be included at the end of the response body.
```

Comment:
:thought_balloon: since the metadata is now outside and after the CAR stream, and @rvagg noted preference for JSON over CBOR, would it be good time to rename this to more self-explanatory `eof=json`? This way we could also support `eof=nil` (removing the need for sending  `{}` when metadata is not needed)

Reply:
How about `meta=json`?

Reply:
> `eof=nil` at some point (removing the need for sending  `{}` when metadata is not needed)

FWIW, if we don't need metadata, then we can omit the entire `meta` (or `eof`) parameter.

Reply:
@bajtos what is your reasoning for preferring `meta` over `eof`?

Reply:
I interpret `meta=json` as "CAR blocks with additional metadata in JSON format". It's open to extension, e.g. allowing metadata blocks in more places than just EOF.

In one of the discussions (sadly not captured in writing), somebody was objecting against using `eof` in `meta=eof` and suggesting to use just `meta` key with no value. I don't remember exact reasons.

On the second thought, considering that we are looking for a short-term fix with least resistance, maybe the key name (`meta` vs `eof`) doesn't matter that much.

Reply:
I like having `json` or `cbor` or other being the value since it makes the metadata options extensible. This means we can start with dag-json as per @rvagg 's suggestion.  For the key, I think that both `eof` and `meta` are not quite right to describe this. Metadata could be placed elsewhere and eof doesnt clear signal that this is metadata. What about `eofmeta=json`?

Reply:
> What about `eofmeta=json`?

Sounds good to me.

Maybe instead of combining `eof` and `meta`, we can use the name `trailer`? E.g. `trailer=json`

Reply:
We are going forward with `meta=eof+json`